lang/sql/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Style Guide for SQL
+pageid: lang-sql
+
+
+Keywords
+--------
+
+All keywords should be uppercase.
+
+    SELECT COUNT(*)
+    FROM someTable
+    WHERE DATEDIFF(NOW(), someTable.timeStamp)
+    ORDER BY ASC
+
+
+Indentation and Whitespace
+--------------------------
+
+Clauses should be separated by newlines.
+
+    SELECT *
+    FROM someTable
+    ORDER BY DESC
+
+When SELECTing multiple columns, and place on the same line.
```

Comment:
I agree with the quotes, but I think it should remain capitalized, as that is how it is used.